### ISSUE TITLE
Set max zoom

### ISSUE THREAD

**Issue by 132ndNeck** [birgersp/dcs-bajas#43]
Would it be possible to set a max zoom level, so that you can only see the world once. Now you can zoom out and it will keep zooming out so you can see several "worlds" with the units in.

**Comment by jkhoel** [comment]
I agree. There is actually a specific setting for this in the map library, so should be a no-brainer
